src/pages/MetricDetail.svelte

```diff
@@ -381,6 +381,28 @@
           />
         </td>
       </tr>
+      {#if pingData.event_monitoring}
+        <tr>
+          <td
+            >Monitoring <HelpHoverable
+              content={"Show real-time monitoring for this event in Mozilla's instance of Looker."}
+            />
+          </td>
+          <td>
+            <div>
+              In the
+              <AuthenticatedLink href={pingData.event_monitoring.base}>
+                Event Monitoring Dashboard
+              </AuthenticatedLink>
```

Comment:
I find linkifying this a bit confusing: we likely only want to showcase the direct link. @Iinh , thoughts?

Reply:
taking a closer look- I'm wondering if it might be a good idea to include this in the Looker row since it's also connected to a Looker dashboard. But there might be a specific reason we'd want to highlight this further by keeping it in a separate row that I'm not aware of? Wdyt?

Reply:
Good question. @scholtzan ?

Reply:
Yeah, I can move it into that row